src/sugar3/graphics/radiotoolbutton.py

```diff
@@ -29,7 +35,13 @@
 
 
 class RadioToolButton(Gtk.RadioToolButton):
-    """An implementation of a "push" button."""
+    '''
+    An implementation of a "push" button. Used with a :class:`RadioPalette` to create a group of 
+    RadioToolButtons.
+    
+    Args:
+        icon_name (string): name of icon to be used
+    '''
```

Comment:
Maybe you should note the names of the gobject props (which you can pass as args to the constructor)